CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ FIND_PACKAGE(LibLZMA REQUIRED)
 
 
 include_directories(SYSTEM ${LibLZMA_INCLUDE_DIRS})
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${LLVM_CXXFLAGS} ${LLVM_VERSION} -fno-strict-aliasing")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${LLVM_CXXFLAGS} ${LLVM_VERSION}")
```

Comment:
Did you check if this is still needed?  I don't think it is, we just turned it on to check if the qemu bug was caused by it.